src/main/java/frc/robot/RobotContainer.java

```diff
@@ -77,7 +99,22 @@ public RobotContainer() {
    * edu.wpi.first.wpilibj.Joystick} or {@link XboxController}), and then passing it to a {@link
    * edu.wpi.first.wpilibj2.command.button.JoystickButton}.
    */
-  private void configureButtonBindings() {}
+  private void configureButtonBindings() {
+    driverController.leftTrigger.whileActiveContinuous(
+        new IntakeCommand(this.intakeSubsystem, IntakeMode.INTAKING)
+            .alongWith(new WristCommand(this.wristSubsystem, WristPosition.INTAKING)));
+    driverController.leftBumper.whileActiveContinuous(
+        new WristCommand(this.wristSubsystem, WristPosition.OUTTAKING)
+            .andThen(new IntakeCommand(this.intakeSubsystem, IntakeMode.OUTTAKING)));
+    operatorController.leftTrigger.whileActiveContinuous(
+        new WristCommand(this.wristSubsystem, WristPosition.INTAKING));
+    operatorController.leftBumper.whileActiveContinuous(
+        new WristCommand(this.wristSubsystem, WristPosition.STOWED));
+    operatorController.rightTrigger.whileActiveContinuous(
```

Comment:
Don't want operator to have intake/outake roller buttons. We just want operator to be able to raise intake up and down

Reply:
oops, this was my bad. sorry simon! i misunderstood the slides and thought "flywheel on/off" was referring to the intake and not the shooter. i have since updated the slides to be clearer. i also updated my discord message where i erroneously asked simon to write this.

![screenshot of the google slides for controls](https://user-images.githubusercontent.com/7608555/196927283-2527aff1-5ab7-49ec-95f9-77b1253cb516.png)